rs/nns/governance/api/src/ic_nns_governance.pb.v1.rs

```diff
@@ -4690,17 +4690,32 @@ impl ProposalRewardStatus {
     }
 }
 
-/// A Request to list minted node provider rewards
+/// A closed range of dates (i.e. includes both start and end dates)
 #[derive(candid::CandidType, candid::Deserialize, serde::Serialize, comparable::Comparable)]
 #[allow(clippy::derive_partial_eq_without_eq)]
-#[derive(Clone, PartialEq)]
-pub struct ListNodeProviderRewardsRequest {}
+#[derive(Debug, Clone, PartialEq)]
+pub struct DateRangeFilter {
+    /// The start date of the range as seconds since epoch
+    pub start_timestamp_seconds: Option<u64>,
+    /// The end date of the range
+    pub end_timestamp_seconds: Option<u64>,
```

Comment:
Should we specify the behavior when `None`?

Reply:
Yes, updating comments.